backend/infrahub/graphql/mutations/proposed_change.py

```diff
@@ -313,58 +313,17 @@ class ProposedChangeMergeInput(InputObjectType):
     id = String(required=True)
 
 
-class ProposedChangeMerge(Mutation):
-    class Arguments:
-        data = ProposedChangeMergeInput(required=True)
-        wait_until_completion = Boolean(required=False)
-
+class ProposedChangeCheckForApprovalRevoke(Mutation):
```

Comment:
From where will this mutation be called? As we don't have any input data would it just revoke all approvals if applicable.

Wondering if this will be an internal function that get's called by a scheduler or some other event?

Reply:
I thought at some point that the frontend might want to call this endpoint each time the corresponding PC page is visited, but I am actually not sure we want this as it may introduce some concurrent/performance issues, I'll remove it.
Scheduling is defined through a cron on corresponding enterprise PR

Reply:
Actually restored it to take an optional list of proposed change ids as input